modules/aws-outposts-environment-info-aws.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-environment-info-aws_{context}"]
+= Obtaining information from your AWS account
+
+You can use the AWS CLI (`aws`) to obtain information from your AWS account.
+
+[TIP]
+====
+You might find it convenient to store some or all of these values as environment variables by using the `export` command.
+====
+
+.Prerequisites
+
+* You have an AWS Outposts site with the required hardware setup complete.
+
+* Your Outpost is connected to your AWS account.
+
+* You have access to your AWS account by using the AWS CLI (`aws`) as a user with permissions to perform the required tasks.
+
+.Procedure
+
+. List the Outposts that are connected to your AWS account by running the following command:
++
+[source,terminal]
+----
+$ aws outposts list-outposts
+----
+
+. Retain the following values from the output of the `aws outposts list-outposts` command:
+
+** The Outpost ID.
+
+** The Amazon Resource Name (ARN) for the Outpost.
+
+** The Outpost availability zone.
++
+[NOTE]
+====
+The output of the `aws outposts list-outposts` command includes two values related to the availability zone: `AvailabilityZone` and `AvailabilityZoneId`. The value you use to configure a compute machine set that creates compute machines in your Outpost is `AvailabilityZone`.
+====
+
+. Using the value of the Outpost ID, show the instance types that are available in your Outpost by running the following command. Retain the values of the available instance types.
++
+[source,terminal]
+----
+$ aws outposts get-outpost-instance-types \
+  --outpost-id <outpost_id_value>
+----
+
+. Using the value of the Outpost ID, show and retain the subnet ID for the Outpost by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 describe-subnets \
+  --filters Name=outpost-id,Values=<outpost_id_value>
```

Comment:
@jeana-redhat  Per [describe-subnets](https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-subnets.html) reference, the filter should be:
```suggestion
  --filters Name=outpost-arn,Values=<outpost_arn_value>
```